app/models/forms/bulk_upload_lettings/guidance.rb

```diff
@@ -4,11 +4,18 @@ class Guidance
       include ActiveModel::Model
       include ActiveModel::Attributes
       include Rails.application.routes.url_helpers
+      include CollectionTimeHelper
 
       attribute :year, :integer
       attribute :referrer
       attribute :organisation_id, :integer
 
+      def initialize(params)
+        super(params)
+
+        self.year = current_collection_start_year if year.nil?
```

Comment:
Annoyingly you can't use the helper method if trying to set a default when defining the attribute (i.e. `attribute :year, :integer, default: current_collection_start_year`), so this is the alternative.